.github/workflows/tests_lgpu_cpp.yml

```diff
@@ -0,0 +1,197 @@
+name: Testing::x86_64::LGPU::C++
+on:
+  workflow_call:
+    inputs:
+      lightning-version:
+        type: string
+        required: true
+        description: The version of Lightning to use. Valid values are either 'release' (most recent release candidate), 'stable' (most recent git-tag) or 'latest' (most recent commit from master)
+      pennylane-version:
+        type: string
+        required: true
+        description: The version of PennyLane to use. Valid values are either 'release' (most recent release candidate), 'stable' (most recent git-tag) or 'latest' (most recent commit from master)
+  release:
+  pull_request:
+    paths:
+      - .github/workflows/tests_lgpu_cpp.yml
+      - pennylane_lightning/core/src/*
+      - pennylane_lightning/core/src/algorithms/**
+      - pennylane_lightning/core/src/bindings/**
+      - pennylane_lightning/core/src/gates/**
+      - pennylane_lightning/core/src/measurements/**
+      - pennylane_lightning/core/src/observables/**
+      - pennylane_lightning/core/src/simulators/base/**
+      - pennylane_lightning/core/src/simulators/lightning_gpu/**
+      - pennylane_lightning/core/src/utils
```

Comment:
Could we have these paths store in a file such as `./.cpp-paths` or a different action and use it in all of these actions recipes?

Reply:
I don't think this is possible, but looking again at the doc I found something to simplify this :)

Reply:
I like the fix! :100: